src/main/java/com/google/ehub/data/FavoriteItemDatastore.java

```diff
@@ -115,6 +125,32 @@ public void removeFavoriteItem(String userEmail, Long itemId) {
     return emails;
   }
 
+  private void addLikeToItem(Long itemId) {
+    try {
+      Entity itemEntity =
+          datastoreService.get(KeyFactory.createKey(ENTERTAINMENT_ITEM_KIND, itemId));
+      itemEntity.setProperty(NUMBER_OF_LIKES_PROPERTY_KEY,
+          (Long) itemEntity.getProperty(NUMBER_OF_LIKES_PROPERTY_KEY) + 1L);
+
+      datastoreService.put(itemEntity);
+    } catch (EntityNotFoundException e) {
+      System.err.println("Entertainment item to add like to doesn't exist: " + e);
+    }
+  }
+
+  private void removeLikeFromItem(Long itemId) {
+    try {
+      Entity itemEntity =
+          datastoreService.get(KeyFactory.createKey(ENTERTAINMENT_ITEM_KIND, itemId));
+      itemEntity.setProperty(NUMBER_OF_LIKES_PROPERTY_KEY,
+          (Long) itemEntity.getProperty(NUMBER_OF_LIKES_PROPERTY_KEY) - 1L);
+
+      datastoreService.put(itemEntity);
+    } catch (EntityNotFoundException e) {
+      System.err.println("Entertainment item to remove like from doesn't exist: " + e);
+    }
+  }
+
```

Comment:
Nit: these methods are pretty similar. We could merge them into one method by adding an extra parameter about whether to add or remove 1.